tests/markFeatureWriter_test.py

```diff
@@ -16,9 +16,10 @@ def test_add_classes(self):
         glyph.appendAnchor(glyph.anchorClass(
             anchorDict={'name': '_bottom', 'x': 100, 'y': 0}))
         lines = []
-        writer = MarkFeatureWriter(
-            ufo, anchorList=(('bottom', '_bottom'),), mkmkAnchorList=(),
-            ligaAnchorList=((('top_1', 'top_2'), '_top'),))
+        writer = MarkFeatureWriter(ufo)
+        writer.anchorList = (('bottom', '_bottom'),)
+        writer.mkmkAnchorList = ()
+        writer.ligaAnchorList = ((('top_1', 'top_2'), '_top'),)
```

Comment:
I'm not sure the change is a clear improvement in terms of ergonomics... [here](https://github.com/google/roboto/blob/bc5156c3114752480ebc8231f31de34a284b3924/scripts/lib/fontbuild/markFeature.py#L20) Roboto just needs to subclass the FeaturesCompiler to put its custom rules in. Whereas here it would need to subclass MarkFeatureWriter as well. That certainly makes it more freestanding, though. wdyt?  
Gosh, what a bikeshed hole.